tiledb/api/src/datatype/mod.rs

```diff
@@ -366,7 +366,7 @@ impl Datatype {
     pub fn is_byte_type(&self) -> bool {
         matches!(
             *self,
-            Datatype::Boolean | Datatype::GeometryWkb | Datatype::GeometryWkt
+            Datatype::Blob | Datatype::GeometryWkb | Datatype::GeometryWkt
```

Comment:
Looks like this changed in core at
```
commit 202930837f8b1ff57fcee954ca0f08009a30858c
Author: Beka Davis <31743465+bekadavis9@users.noreply.github.com>
Date:   Mon Mar 25 15:54:30 2024 -0400
```

So at the time it was written, it may well have been correct!

That's the trouble with having to replicate some of this logic from core in this layer, I suppose.

Reply:
Well it was never correct, but we missed finding a bug in core by ~12 days.

Reply:
Does "correct" mean "does the right thing in principle" or "does what core does"?
:)